lib/src/display/render_loop.dart

```diff
@@ -62,7 +74,20 @@ class RenderLoop extends RenderLoopBase {
     }
 
     for (int i = 0; i < _stages.length; i++) {
-      _stages[i].materialize(_currentTime, deltaTime);
+      if(_stages[i].renderMode != StageRenderMode.AUTO_DIRTY || juggler.hasAnimatables || _stages[i].juggler.hasAnimatables || _stages[i].dirty){
```

Comment:
Issue: `Juggler.hasAnimatables` returns `false` after `Juggler.advanceTime`, leading to `materialize`not being invoked even though Juggler would need one more frame that runs materialize.
@bp74 any ideas how to circumvent this?